src/styles/_catalog.scss

```diff
@@ -15,8 +11,25 @@
     border-radius: 5px;
     margin-bottom: 48px;
   }
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    display: grid;
```

Comment:
move medias into the catalog selector. and don't set `display: grid;` each time, just set it once for main catalog selector